controllers/qrCodeAuth.js

```diff
@@ -0,0 +1,26 @@
+const qrAuthQuery = require("../models/qrCodeAuth");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+const updateAuthStatus = async (req, res) => {
+  try {
+    const userId = req.userData.id;
+    const authStatus = req.params.authorization_status;
+    const result = await qrAuthQuery.updateStatus(userId, authStatus);
+
+    if (result.userExists) {
```

Comment:
I would suggest following the fail fast principle and add a check for `!result.userExisits` and returning 404
Otherwise, return success response outside `if` block
It also improves the readability of code